README.md

```diff
@@ -98,7 +98,17 @@ As a contributor, you can also point to your own fork containing the `tldr.zip`
 
 ```js
 {
-  "repository" : "http://myrepo/assets/tldr.zip",
+  "repository" : "http://myrepo/assets/tldr.zip"
+}
+```
+
+If you want to prevent that a cache update is performed each time no page is found for a command, then
+you can set configuration variable `skipUpdateWhenPageNotFound` to the value `true` 
+(the default value is `false`):
```

Comment:
```suggestion
If you want to prevent that a cache update is performed each time no page is found for a command, then
you can set the configuration variable `skipUpdateWhenPageNotFound` to `true` (defaults to `false`):
```